bokehjs/src/coffee/core/layout/layout_canvas.coffee

```diff
@@ -0,0 +1,49 @@
+_ = require "underscore"
+
+{Variable, EQ, GE, Strength}  = require "./solver"
+Model = require "../../model"
+p = require "../properties"
+
+class LayoutCanvas extends Model
+  type: 'LayoutCanvas'
+
+  @internal {
```

Comment:
I presume (based on other changes) this is an internal class. If so, it should be marked accordingly.